services/ux-backend/main.go

```diff
@@ -0,0 +1,28 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+	"strconv"
+
+	handler "github.com/red-hat-storage/ocs-operator/v4/services/ux-backend/handlers"
+	"k8s.io/klog/v2"
+)
+
+func main() {
+
+	klog.Info("Starting ux backend server")
+
+	http.HandleFunc("/onboarding-tokens", handler.OnboardingTokensHandler)
+
+	listenPort, err := strconv.Atoi(os.Getenv("UX_BACKEND_PORT"))
+	if err != nil {
+		listenPort = 8080
+	}
+	klog.Info("ux backend server listening on port ", listenPort)
```

Comment:
Please log differentiate between an empty env var, where it makes sense to default to 8080, and one that has an invalid port, where it makes sense to report and error and shutdown.
```suggestion
	var listenPort int;
	listenPortAsString = os.Getenv("UX_BACKEND_PORT")
	if listenPortAsString == "" {
		klog.Infof("No user-defined server listening port provided, defaulting to %d, defaultListeningPort)
		listenPort = defaultListeningPort
	} else if listenPort, err :=  strconv.Atoi(listenPortAsString); err != nil {
		klog.Errorf("Malformed user-defined listening port: %s. shuting down", listenPortAsString)
		os.Exit(-1)
	}
```

Reply:
right make sense